doc/classes/DisplayServer.xml

```diff
@@ -1178,11 +1178,22 @@
 		<method name="status_indicator_set_icon">
 			<return type="void" />
 			<param index="0" name="id" type="int" />
-			<param index="1" name="icon" type="Image" />
+			<param index="1" name="icon" type="Texture2D" />
 			<description>
 				Sets the application status indicator icon.
 			</description>
 		</method>
+		<method name="status_indicator_set_menu">
+			<return type="void" />
+			<param index="0" name="id" type="int" />
+			<param index="1" name="menu_rid" type="RID" />
+			<description>
+				Sets the application status indicator native popup menu.
+				[b]Note:[/b] On macOS, menu is activated by any mouse button, activation callback is not triggered.
+				[b]Note:[/b] On Windows, menu is activated by right mouse button, selecting icon and pressing Shift+F10 or the applications key, activation callback for the other mouse buttons is still triggered.
+				[b]Note:[/b] Native popup is supported only if [NativeMenu] supports [constant NativeMenu.FEATURE_POPUP_MENU] feature.
```

Comment:
```suggestion
				[b]Note:[/b] On macOS, the menu is activated by any mouse button. Its activation callback is [i]not[/i] triggered.
				[b]Note:[/b] On Windows, the menu is activated by the right mouse button, selecting the status icon and pressing [kbd]Shift + F10[/kbd], or the applications key. The menu's activation callback for the other mouse buttons is still triggered.
				[b]Note:[/b] Native popup is only supported if [NativeMenu] supports the [constant NativeMenu.FEATURE_POPUP_MENU] feature.
```
... What's the "selecting icon" and "applications key" though?

Reply:
> "applications key" though?

This thing (it's referred in the Windows docs as "applications key").
![appkey](https://upload.wikimedia.org/wikipedia/commons/3/37/Menu_key_--_Diatec_and_Dell.jpg)

> What's the "selecting icon"

You can select tray icons with keyboard (using Windows + B, arrow keys and Enter).

Reply:
You learn something new every day... It may be worth mentioning the `KEY_*` constant equivalent if it does exist.

Reply:
Interesting, I always thought this was the context key.